modules/core/test/v2/unit/ethWallet.ts

```diff
@@ -265,6 +265,220 @@ describe('Ethereum Hop Transactions', co(function *() {
   }));
 }));
 
+describe('Ethereum batch transaction data validation', function() {
+  const bitgo = new TestV2BitGo({ env: 'test' });
+  bitgo.initializeTestVars();
+  const coin = bitgo.coin('teth');
+
+  const batcherContractAddress = common.Environments[bitgo.env].batcherContractAddress;
+
+  const initialParams = {
+    recipients: [{
+        address: '0x1111111111111111111111111111111111111111',
+        amount: '5'
+      }, {
+        address: '0x2222222222222222222222222222222222222222',
+        amount: '5'
+      }
+    ]
+  };
+
+  const buildBadParams = (badRecipients) => {
+    const badPrebuildParams = {
+      recipients: badRecipients
+    };
+
+    return {
+      txParams: initialParams,
+      txPrebuild: badPrebuildParams
+    };
+  };
+
+  const runBadBatch = co(function *(badRecipients, expectedErrMsg) {
+    const badParams = buildBadParams(badRecipients);
+    let errMsg = '';
+    let threw = false;
+
+    try {
+      yield coin.verifyTransaction(badParams);
```

Comment:
You can replace this whole try/catch business with `yield coin.verifyTransaction(badParams).should.be.rejectedWith(expectedErrMsg);`